### DISCUSSION TITLE
Trouble deploying Agama Project

### DISCUSSION THREAD

**Discussion by pdxwebdev** [JanssenProject/jans#7982] [General]
Hello,

I am just getting started with Janssen and I'm following this guide https://docs.jans.io/v1.0.22/admin/recipes/inbound-oidc/

I'm trying to deploy and test my .gama flow.

I'm receiving this error when executing the command shown the documentation:
```
config-cli.py --operation-id post-config-scripts --host ******* --client-id ********* --client-secret *********
No Operation ID post-config-scripts was found.
```

ps. added the --host param because I was receiving an error related to it being blank, probably not related by thought I'd mention it.

Let me know any other information you need in order to help, happy to provide.

Thanks!

**Comment by nynymike** [comment]
If you're new to Janssen it's probably better to use the TUI to deploy your projects. 

1. Start tui

```shell
/opt/jans/jans-cli/config-cli-tui.py
```

2. Menu navigate to Auth Server / Agama and select `Upload Projects`

![image](https://github.com/JanssenProject/jans/assets/3717101/172f0f83-6ec8-4c3b-93f0-00cba198d57c)

3. Select .gama file

![image](https://github.com/JanssenProject/jans/assets/3717101/87ff6df2-340c-463f-8c7c-a9b2ace5a898)

4. Wait a minute or so for the project to load... tab around the TUI and check out some other stuff while your waiting... Then check back to see if the Agama Project is loaded.

**Reply by pdxwebdev** [reply]
Thanks for the reply. I had already done the deploy through TUI as you suggest. That went fine. But then I'm trying to test using the tarp extension and not seeing openid in available acr options. So that's what I thought the guide was helping with.

**Comment by nynymike** [comment]
Acr would be `agama` ... Use Add'l parameters `{"agama_flow":"org.gluu.inbound.openid"}`

**Reply by pdxwebdev** [reply]
![Screenshot from 2024-03-07 09-03-04](https://github.com/JanssenProject/jans/assets/490176/23c91a00-bbbf-494b-bab0-ac032267e30d)
This is what I see in acr values. Should there be an agama option in acr values?

**Reply by nynymike** [reply]
Because you have not enabled the agama script.  In Jans Auth Server, each `acr` is mapped to a specific [Jans Person Authn Script](https://docs.jans.io/v1.0.22/admin/developer/scripts/person-authentication/).

![image](https://github.com/JanssenProject/jans/assets/3717101/040b69fe-a5d0-4ea1-a939-16ddbb64943e)

BTW, Tarp renders the select list based on the `acr_values_supported` published in `.well-known/openid_configuration`

**Reply by pdxwebdev** [reply]
Ahhh, I had to search it in the list of scripts. Ok, unblocked now. Thanks!

**Reply by nynymike** [reply]
Search the scripts for `agama` next time!
